coalib/bears/Bear.py

```diff
@@ -298,6 +298,10 @@ def execute(self, *args, debug=False, **kwargs):
                     not self.BEAR_DEPS):
                 del kwargs['dependency_results']
 
+            # Don't pass `aspects` if the bear doesn't support it.
+            if not hasattr(self, 'aspects') and 'aspects' in kwargs:
+                del kwargs['aspects']
```

Comment:
With `kwargs.pop('aspects', None)` you don't need to check for `'aspects' in kwargs`